### DISCUSSION TITLE
一次购买多张卡密的BUG

### DISCUSSION THREAD

**Discussion by chrome007** [Baiyuetribe/kamiFaka#112] [Bug反馈] [answered]
你好，发现BUG，用户一次购买多张卡密（一个订单），系统会发送相应数量卡密，
但是后台已经发送的卡密依旧为“有效”状态，而不是“失效”。
这就会造成下个用户再购买，会发送已经“失效”卡密。
看了帮助文档，和翻遍了后台设置，没有找到相应解决策略。

**Comment by Baiyuetribe** [comment, marked as answer]
该问题早期版本存在，1.8最新版已经修复过，不知道你现在用的哪个

**Reply by chrome007** [reply]
你好，大佬，是使用的最新版本1.8中发现的该问题。
1.8中依然存在该问题。
后台：
程序详情
最新版本：V1.8
